Formula/lysp.rb

```diff
@@ -1,7 +1,7 @@
 class Lysp < Formula
   desc "Small Lisp interpreter"
-  homepage "http://www.piumarta.com/software/lysp/"
-  url "http://www.piumarta.com/software/lysp/lysp-1.1.tar.gz"
+  homepage "https://web.archive.org/web/20181230044600/www.piumarta.com/software/lysp/"
+  url "https://web.archive.org/web/20160503234302/www.piumarta.com/software/lysp/lysp-1.1.tar.gz"
```

Comment:
Unmaintained, requires heavy patching (and non system compiler) to build. Let's remove it from core.